### PR TITLE
Fix sigterm handling

### PR DESCRIPTION
**Pull request by luebken**
Bugfix: save server in own variable so it can be used in SIGTERM handling

### CODE REVIEW

**Comment by luebken** [comment]
RFR @ZeissS @zyndiecate @htr @marians

**Comment by marians** [comment]
Interesting. I'll take this as an information. Can't review this.

File: server.js [hunk 1]
```diff
@@ -35,9 +35,12 @@ http.createServer(function (request, response) {
...
     }
   });
-}).listen(httpPort, httpAddress);
+})
+
+server.listen(httpPort, httpAddress);
 
 process.on('SIGTERM', function () {
+  console.log("Received SIGTERM. Exiting.")
   server.close(function () {
```
Comment (zyndiecate) [inline server.js]: Where did this `server` come from before? Looks like there was none, means it was really "broken".

**Comment by zyndiecate** [comment]
Can you confirm the process is exiting with `0` when it receives `SIGTERM`?

**Comment by luebken** [comment]
Yes: 
Terminal 1:
```
± |master ✗| → docker ps
CONTAINER ID        IMAGE                                                  COMMAND                CREATED             STATUS              PORTS                    NAMES
c5e79551dcc6        registry.giantswarm.io/luebken/currentweather:latest   "/nodejs/bin/node se   15 seconds ago      Up 14 seconds       0.0.0.0:1337->1337/tcp   stoic_bartik
4043878a4b5d        redis:latest                                           "/entrypoint.sh redi   22 seconds ago      Up 21 seconds       6379/tcp                 redis

 LuebkenAir in ~/workspace/giantswarm/giantswarm-firstapp-nodejs
± |fix-sigterm-handling ✓| → docker stop c5e79551dcc6
c5e79551dcc6
````

Terminal 2:
```
docker run --link redis:redis -p 1337:1337 -ti --rm registry.giantswarm.io/luebken/currentweather
Server running at http://<ip-address>:1337/

Received SIGTERM. Exiting.
```

Will merge.

File: server.js (hunk 1 above)
Comment (luebken) [inline server.js]: Yes. Null pointer. Really bad.

**Comment by zyndiecate** [comment]
I would have loved to see the result of `echo $?` in terminal 2.

**Comment by luebken** [comment]
echo $?
0

**Pull request merged**